eradiate/data/__init__.py

```diff
@@ -40,12 +42,14 @@
 import xarray as xr
 
 from .absorption_spectra import _AbsorptionGetter
+from .profiles import _ProfilesGetter
 from .solar_irradiance_spectra import _SolarIrradianceGetter
 from .spectral_response_function import _SpectralResponseFunctionGetter
 from .. import path_resolver as _presolver
 
 _getters = {
     "absorption_spectrum": _AbsorptionGetter,
+    "profiles": _ProfilesGetter,
```

Comment:
Same here: please use `atmospheric_profiles`.

Reply:
After our discussion, I went for `thermoprops_profiles`.